clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/description/DeployCloudFoundryServiceDescription.java

```diff
@@ -40,10 +41,13 @@
   @JsonIgnore
   private ServiceAttributes serviceAttributes;
 
+  @JsonIgnore
+  private UserProvidedServiceAttributes userProvidedServiceAttributes;
+
   @Data
   public static class ServiceAttributes {
-    String service;
     String serviceName;
+    String service;
```

Comment:
How is this distinct from service name and service plan? Does this effectively make `ServiceAttributes` a union type?

Reply:
`ServiceAttributes` are used specifically to create services from the market place.
`UserProvidedServiceAttributes` are used specially to create user provided services.

`UserProvidedServiceAttributes` don't specify the following 
`service`: name of the service from the market place or `servicePlans`

I was hoping to keep `ServiceAttributes` distinct from `UserProvidedServiceAttributes` because most of the values used in `UserProvidedServiceAttributes` are not used in `ServiceAttributes`